### ISSUE TITLE
[Bug]: Routes are not auto completed and not clickable

### ISSUE THREAD

**Issue by schonhoff** [laravel-idea/plugin#1026]
### Bug description

Hello,

after changing our project to the new Laravel 11 skeleton we have a problem with auto completing route names. We can't even click on them. 

Here is an example route:
![image](https://github.com/user-attachments/assets/5a0cd564-a8bf-443d-b20c-65da43b0cadd)

The route is in an extra route file called 'reg-office-api'.
![image](https://github.com/user-attachments/assets/98541716-5e39-49b7-9e49-f799a0007d05)

This route file is registered in the `bootstrap/app.php` like this:
![image](https://github.com/user-attachments/assets/5eb0b8b9-193a-4a62-8ca1-731e8702d09c)

I can't even find a routes file in the _laravel_idea directory. If you need more information please let me know. I'm trying to give as much information as possible.

### Plugin version

8.2.1.241

### Operating system

Windows

### Steps to reproduce

1.) Set up a new Laravel Application
2.) Add a new route file to the routes directory
3.) Add a routes group with an `as` function in the `then:` parameter in the `bootstrap/app.php` file
4.) Add a route into the routes file
5.) Try to auto complete the file inside a controller

**Comment by schonhoff** [comment]
Hi,

after I checked out the documentation and tried some things out, I found that the

```php
/**
 * @routePrefix("reg-office-api.order.")
 */
```
helped. I use the standard Laravel definition of routes (seen here in the second code block https://laravel.com/docs/11.x/routing#routing-customization ) and it worked on Laravel 10 RouteServiceProvider. After switching to the app.php skeleton it won't work and I need to include the docBlock mentioned above. Maybe this "bug" can be changed to a "feature request" because an auto completion on a standard new Laravel app with the slim skeleton would be awesome (if you are using the standard mentioned in the docs).
